README.md

```diff
@@ -150,11 +150,13 @@ If you are using a macOS runner with an Apple silicon processor, verify that Jav
 >**Tip:** One convenient way to include the required dependencies on a self-hosted runner is to specify the [MATLAB Dependencies container image on Docker&reg; Hub](https://hub.docker.com/r/mathworks/matlab-deps) in your workflow.
 
 #### Licensing
-Product licensing for your workflow depends on your project visibility as well as the type of products to set up:
+Product licensing for your workflow depends on your project visibility as well as the types of products the workflow uses:
+
+- Public project — The action automatically licenses all products for you, except for transformation products, such as MATLAB Coder and MATLAB Compiler.
```

Comment:
Is it safe to say, "The **Setup MATLAB** action?" Is this the action that performs the licensing?

Reply:
The Setup MATLAB action is not the action that performs licensing. It's actually the Run* actions that handle licensing.

Reply:
Does the clarification help?